crawl-ref/source/dat/des/branches/swamp.des

```diff
@@ -1260,6 +1285,175 @@ tt... PppPqqq    QqqQQqQqqwwwwww.t
            PPPP    PPPP       ttt
 ENDMAP
 
+# The various "reclaimed" decor vaults just place a pattern of rock walls over
+# the terrain, as if the swamps and forest had formed around the ruins.
+# Sometimes they end up barely visible, enclosed entirely in mangroves except
+# for a corner or something. This is acceptable.
+NAME:  nicolae_swamp_reclaimed_barracks
+DEPTH: Swamp
+TAGS:  extra no_exits luniq_nicoreclaimed
+SUBST: x = x:50 _
+CLEAR: _
+MAP
+xxxxxxxxxxxxxxxxxxx
+x  x  x  x  x  x  x
+x  x  x  x  x  x  x
+x                 x
+                  x
+                  x
+x                 x
+x  x  x  x  x  x  x
+x  x  x  x  x  x  x
+xxxxxxxxxxxxxxxxxxx
+ENDMAP
+
+NAME:  nicolae_swamp_reclaimed_columns
+DEPTH: Swamp
+TAGS:  extra no_exits luniq_nicoreclaimed
+SUBST: x = x:50 _
+CLEAR: _
+MAP
+xx  xx  xx  xx  xx  xx  xx
+xx  xx  xx  xx  xx  xx  xx
+_
+_
+xx                      xx
+xx                      xx
+      xx  xx  xx  xx
+      xx  xx  xx  xx
+xx                      xx
+xx                      xx
+      xx  xx  xx  xx
+      xx  xx  xx  xx
+xx                      xx
+xx                      xx
+_
+_
+xx  xx  xx  xx  xx  xx  xx
+xx  xx  xx  xx  xx  xx  xx
+ENDMAP
+
+NAME:  nicolae_swamp_reclaimed_house
```

Comment:
These vaults are cool and I like the concept, but I feel you need to scale down their size and test their placements to see that they're actually placing in a meaningful way. Tests with this vault, which isn't the largest one, show that it rarely places such that even 25% of the vault is visible, and even then it's pretty difficult to see the overall structure, which is a shame. It might be the scaling down the vaults to 66-50% of their preset size will be enough, but you might want to test placements of them individually to know which need this and to what degree.